core/os/device/device.proto

```diff
@@ -173,6 +173,8 @@ message Configuration {
   Drivers drivers = 4;
   // Perfetto capability.
   PerfettoCapability perfetto_capability = 5;
+  // Angle package name
+  string angle_package = 6;
```

Comment:
Not sure if this is still relevant. If you want to restore the angle package after tracing, this can be a custom message type similar to configuration, where it has the list of all the packages and whichever was enabled before taking the trace and so on. This can later be used to set and unset the package to be used for tracing.

Reply:
This is currently where I store the detected ANGLE package name in device.go. Then, if ANGLE tracing mode is selected, I verify that we stored the detected ANGLE package in adb.go before enabling ANGLE.
So I could kill this but then I'd have to find a different place to store ANGLE package name.